components/caseStudyDetails/CaseStudiesDetailHeadings.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { motion } from "framer-motion";
+
+import { paragraphVariants, headingVariants } from "@/utils";
+
+const CaseStudiesDetailHeadings = ({
+  smallText,
+  headingText,
```

Comment:
Did you consider moving this into your utils, with paragrahpVariants?

Reply:
![Screenshot 2023-09-28 at 1 11 38 PM](https://github.com/DevTaehong/portfolio_jsm/assets/71358207/ef755538-f380-40c4-8e81-bffcfe9337d9)
I'm sorry. I don't understand. Did you mean to move this component into my utils?? Isn't this a React component?